tests/test_config.py

```diff
@@ -74,10 +74,11 @@ def test_named_config_file(self, file_class: FilePathType) -> None:
         assert not cov.config.branch
         assert cov.config.data_file == "delete.me"
 
-    def test_toml_config_file(self) -> None:
-        # A pyproject.toml file will be read into the configuration.
+    @pytest.mark.parametrize("filename", ["pyproject.toml", ".coveragerc.toml"])
+    def test_toml_config_file(self, filename) -> None:
+        # A pyproject.toml and .coveragerc.toml file will be read into the configuration.
```

Comment:
Well, not both, just one of them, right?
```suggestion
        # A pyproject.toml or .coveragerc.toml file will be read into the configuration.
```